build/Dockerfile

```diff
@@ -112,8 +112,11 @@ RUN git clone https://github.com/foundation-model-stack/fms-hf-tuning.git && \
 RUN mkdir -p /licenses
 COPY LICENSE /licenses/
 
-COPY launch_training.py /app
-RUN chmod +x /app/launch_training.py
+# Copy scripts and default configs
+COPY launch_training.py accelerate_launch.py accelerate_fsdp_defaults.yaml /app/
```

Comment:
@anhuong  Since, `accelerate_fsdp_defaults.yaml` not in the current directory where Dockefile is present the provided image build command in the docs would fail isn't it?

https://github.com/foundation-model-stack/fms-hf-tuning/pull/98/files#diff-4b71ec736aee936feb417e5193e800a1014a8e2df2a5b5b92fd78fbd3d781dbdR71-R74

Reply:
Yes you are both correct. When I run my image build I have to explicitly copy the file over but I think I have a way to fix this....

Reply:
This thread is resolved.
cc: @Ssukriti